test/jasmine/tests/geo_test.js

```diff
@@ -1216,7 +1216,7 @@ describe('Test geo interactions', function() {
                     }
 
                     gd.calcdata = undefined;
-                    Plotly.plot(gd);
+                    Plotly.newPlot(gd, gd.data, gd.layout);
```

Comment:
At some point we recommended this pattern - `gd.calcdata = undefined` before a `Plotly.plot` call - if you wanted to update the figure object in place and let the graph know it should redo its `calc` step.

If you replace `plot` with `newPlot` in these cases, you're doing too much cleaning so we're not testing the intended behavior.

Instead, these cases should turn into `Plotly.react` with a `layout.datarevision` value that changes every time (set its value to `i` perhaps). And you can remove the `gd.calcdata = undefined`.

Reply:
Thanks for the info. 
Addressed in https://github.com/plotly/plotly.js/pull/5393/commits/981cc40fe02469db4bfa2ffe3de8122401733aff.